src/pages/staking-hub/onboarding/step5/0addNode.tsx

```diff
@@ -48,7 +49,22 @@ export default function AddNode() {
             label: 'node',
           },
         }),
-      ).unwrap();
+      ).unwrap()
+        .catch(e => {
+          if (e.error == "Safe= does not exist or it's still not indexed") {
```

Comment:
The error will not get tiggered as e.error will not be "Safe= does not exist or it's still not indexed"
after = in the error you get extra text.
I would use includes('does not exist or it's still not indexed')